mimic/model/clb_objects.py

```diff
@@ -78,6 +104,14 @@ def add_load_balancer(self, tenant_id, lb_info, lb_id, current_timestamp):
 
         return {'loadBalancer': new_lb}, 202
 
+    def set_attributes(self, lb_id, kvpairs):
+        """
+        Sets zero or more attributes on the load balancer object.
+        Currently supported attributes include: status.
+        """
+        if "status" in kvpairs:
```

Comment:
I had intended the REST handler to perform input validation for this call, but I see the point.  Changes shortly.